microsoft-edge/extensions-chromium/getting-started/part1-simple-extension.md

```diff
@@ -16,7 +16,13 @@ The goal for this tutorial is to build a Microsoft Edge extension, starting with
 *   Adding icons.
 *   Opening a default pop-up dialog.
 
-If you want to test out the completed extension that you are building in this tutorial, download the source code from [MicrosoftEdge-Extensions repo > extension-getting-started-part1](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part1/part1). The source code has been updated from Manifest V2 to Manifest V3.
+If you want to test out the completed extension that you are building in this tutorial, download the source code from [MicrosoftEdge-Extensions repo > extension-getting-started-part1](https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples/extension-getting-started-part1/part1).  The source code uses Manifest V3.
```

Comment:
has repo been updated from MV2 to MV3 yet?
re: part 1 & 2 .md PRs.
https://github.com/microsoft/MicrosoftEdge-Extensions/tree/main/Extension%20samples
last commit = 2 years ago.
has repo been updated from MV2 to MV3?
did repo have MV2 ever?
does repo now have MV3?
@bhuvanapriyap

Reply:
The source code on the repo seems to be using V2 at the moment. We need to update it at the same time as we update the docs, so as not to cause confusion.

Reply:
Extensions repo part 1 still says mv2:
https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/Extension%20samples/extension-getting-started-part1/part1/manifest.json
"manifest_version": 2,

Extensions repo Part 2 already says MV3, and I think never said MV2:
https://github.com/microsoft/MicrosoftEdge-Extensions/blob/main/Extension%20samples/extension-getting-started-part2/extension-getting-started-part2/manifest.json
"manifest_version": 3,

Reply:
Closing, might create fresh "todo" PR Comment to update Extensions repo Part 1 from mv2 to mv3.